app/controllers/core_induction_programme/lessons_controller.rb

```diff
@@ -1,7 +1,36 @@
 # frozen_string_literal: true
 
 class CoreInductionProgramme::LessonsController < ApplicationController
+  include Pundit
+
+  after_action :verify_authorized, except: :show
+  before_action :authenticate_user!, except: :show
+
   def show
     @course_lesson = CourseLesson.find(params[:lesson_id])
   end
+
+  def edit
+    @lesson = CourseLesson.find(params[:lesson_id])
+    authorize @lesson
+
+    @lesson_preview = params[:lesson_preview] || @lesson.content
+    @course_lesson = Govspeak::Document.new(@lesson_preview, options: { allow_extra_quotes: true }).to_html
+  end
+
+  def update
+    @lesson = CourseLesson.find(params[:lesson_id])
+    authorize @lesson
+
+    respond_to do |format|
```

Comment:
We are only rendering html so you can take out respond_to do |format| everywhere really.

Reply:
Just render and redirect, you don't need the format.html